### ISSUE TITLE
Decouple transmission of report shares from leader to helper from aggregation job creation

### ISSUE THREAD

**Issue by tgeoghegan** [ietf-wg-ppm/draft-ietf-ppm-dap#409]
Currently, the leader transmits encrypted report shares to the helper inline in an `AggregationJobInitReq` message. This is inefficient in the face of VDAFs that allow querying the same batch of reports multiple times with different aggregation parameters. Given the current protocol, those report shares must be retransmitted from leader to helper each time. This could be a significant waste of bandwidth, depending on the size of report shares and the number of collections that will be run against a batch.

**Comment by tgeoghegan** [comment]
This is akin to #405, and corresponds to [an `OPEN ISSUE` that's been in the protocol text for a while](https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/blob/89e5dc32be05ebd4eb032c7f6d0aaa7074c69b08/draft-ietf-ppm-dap.md?plain=1#L1180).

**Comment by tgeoghegan** [comment]
Solving this correctly is tricky: we don't want to force the leader to keep track of whether it has ever transmitted any report share to the helper, and in any case, we have to account for the possibility that the helper has garbage collected a report share between two runs of the collect protocol on a batch.

**Comment by tgeoghegan** [comment]
Turns out this is the same as #230.

**Comment by tgeoghegan** [comment]
I conflated the questions of decoupling _report share_ transmission and _aggregation parameter_ transmission when I closed this. This issue is still relevant so I'm reopening it. This is particularly relevant in the context of Poplar1, or other VDAFs where the same reports may be aggregated multiple times. That means we need to study the performance of Poplar1 to understand whether we want to do this, especially as we are otherwise trying to converge DAP towards a stable state.
